name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/WidgetFactory.java

```diff
@@ -215,6 +215,8 @@ public enum WidgetFactory
 
     HEATMAP_EARNINGS(Messages.LabelHeatmapEarnings, Messages.LabelEarnings, EarningsHeatmapWidget::new),
 
+    EARNINGS_DIVIDEND_LIST(Messages.LabelEarningsDividendList, Messages.LabelEarnings, DividendListWidget::new),
```

Comment:
The name is very generic. And "earnings" and "dividends" is kind of duplicated - dividends are earnings, but not all earnings are dividends. Why not call it UPCOMING_DIVIDENDS_LIST. And the class UpcomingDividendsListWidget?

Reply:
When reading the code I had the impression that "EARNINGS" was use as a kind of "grouping mechanism" in order to see widgets that belong together being sorted together in the Outline. I can rename the enum-name if really needed but I'd keep the class name as it is.

Reply:
I've renamed the enum. Better now?